iOS_Learning/Services/SignUpService/Model/SignUpResponse.swift

```diff
@@ -0,0 +1,19 @@
+//
+//  SignUpResponse.swift
+//  iOS_Learning
+//
+//  Created by Vlad_Rosca on 27.03.2023.
+//
+
+import Foundation
+
+struct SignUpResponse: Decodable {
+    let email: String
+    let password: String
+    let name: String
+    let avatar: String
+    let role: String
```

Comment:
From what I read this should be a struct as they are easier to manipulate throuh the app (e.g. it is easier to map into other objects - like an app/presentation layer object ).

Reply:
In case you have a fixed amount of values you should use Enum, please read more [here](https://docs.swift.org/swift-book/documentation/the-swift-programming-language/enumerations/).